### PR TITLE
removing props (bugged, repeated, unecessary, etc)

### PR DESCRIPTION
**Pull request by jak3z**
this pull is more to remove and warn me of certain objects that don't make sense to me or that are wrong, I probably won't fix it, just remove

### CODE REVIEW

File: modloader/Project Props/data/LSprops.ipl [hunk 1]
```diff
@@ -3119,7 +3119,6 @@ inst
 19248, paper2, 0, 1858.8999, -1875.24988, 12.4594, 0, 0, 0.537299, 0.843392, -1
 19248, paper2, 0, 1730.41016, -1944.91003, 12.56, 0, 0, 0.986286, -0.165046, -1
 19249, paper3, 0, 2124.5, -1788.85999, 12.5359, 0, 0, 0.970296, -0.24192, -1
-1582, pizzabox, 0, 2102.12012, -1795.93982, 13.25, 0, 0, 0, 1, -1
```
Comment (jak3z) [inline modloader/Project Props/data/LSprops.ipl:3122]: floating object

File: modloader/Project Props/data/LSprops.ipl [hunk 2]
```diff
@@ -5219,7 +5218,6 @@ inst
 19293, weed9, 0, 1902.73511, -1145.77417, 23.41209, 0.0217972, -0.0361525, 0.998725, 0.0276686, -1
 19293, weed9, 0, 1893.52808, -1186.61267, 21.78627, -0.0381945, 0.0430293, -0.545102, 0.836393, -1
 19293, weed9, 0, 1979.90356, -1165.32178, 19.45752, -0.0183248, 0.0239953, -0.705156, 0.708409, -1
-19293, weed9, 0, 1876.00439, -1251.29614, 24.17529, -0.00251213, 0.00218332, 0.655981, 0.75477, -1
```
Comment (jak3z) [inline modloader/Project Props/data/LSprops.ipl:5222]: floating object

File: modloader/Project Props/data/LSprops.ipl [hunk 3]
```diff
@@ -9319,7 +9317,6 @@ inst
 1810, CJ_FOLDCHAIR, 0, 2463.95972, -1011.49988, 59.1172, 0, 0, -0.0146602, 0.999893, -1
 1810, CJ_FOLDCHAIR, 0, 2524.17993, -1655.35999, 14.4906, 0, 0, 0.380748, 0.924679, -1
 1810, CJ_FOLDCHAIR, 0, 2334.76001, -1044.56995, 51.3537, 0, 0, 0.877062, -0.480376, -1
-2714, CJ_OPEN_SIGN_2, 0, 2068.40991, -1777.80994, 14.2281, 0, 0, 0.707106, 0.707107, -1
```
Comment (jak3z) [inline modloader/Project Props/data/LSprops.ipl:9322]: repeated and wrong position and rotation

File: modloader/Project Props/data/LSprops.ipl [hunk 4]
```diff
@@ -9370,7 +9367,6 @@ inst
 17518, telwire_01_LAe2, 0, 2361.94019, -1699.93982, 15.9219, 0, 0, 0, 1, -1
 1306, tlgraphpolegen, 0, 2233.87964, -991.57117, 65.5796, 0, 0, 0.276979, 0.960876, -1
 14604, tv_stand_bike, 0, 2408.86011, -1548.84998, 24.15, 0, 0, 0.101056, 0.994881, -1
-3497, vgsxrefbballnet2, 0, 2055.59009, -1694.67004, 16.3609, 0, 0, 0.707106, 0.707107, -1
```
Comment (jak3z) [inline modloader/Project Props/data/LSprops.ipl:9373]: the house of big smoke already have

File: modloader/Project Props/data/LSprops.ipl [hunk 5]
```diff
@@ -10783,7 +10779,6 @@ inst
 1457, DYN_OUTHOUSE_2, 0, 1857.73999, -2110.8999, 15.7469, 0, 0, 0, 1, -1
 17527, gangblock1Tr_LAe, 0, 2414.40015, -1362.20007, 32.54, 0, 0, 0, 1, -1
 3648, ganghous02_LAx, 0, 2394.66968, -1311.20007, 27.0239, 0, 2.6226e-06, 0, 1, -1
-9321, garage_sfn01, 0, 2473.46167, -1709.75208, 13.9859, 0, 0, -0.707108, 0.707105, -1
```
Comment (jak3z) [inline modloader/Project Props/data/LSprops.ipl:10786]: i dont think if is good and is weird

**Comment by user-grinch** [comment]
It'd be better if you push a commit after making full changes next time (8 commits within a span of few hours for LSprops.ipl). Also, I believe Zeneric already made a lot of changes to his LSprops.ipl which aren't merged to the repo yet. So can't say how it would be mergeable later on. I'll send him the changes, the rest is on him.

Anyway, I've checked some of them, looks good. Let me know when you're done.

**Comment by jak3z** [comment]
> It'd be better if you push a commit after making full changes next time (8 commits within a span of few hours for LSprops.ipl).
oh sorry, it's because I'm doing it on my fork and every change I make is going to pull so I'll close it for now
> So can't say how it would be mergeable later on. I'll send him the changes, the rest is on him.
okay!

**Pull request closed without merging**